features/step_definitions/wait_steps.rb

```diff
@@ -1,5 +1,9 @@
 # frozen_string_literal: true
 
+When(/^it is configured to raise exceptions for wait fors$/) do
```

Comment:
Lets make this a Given step and re-order the cukes accordingly.
`Given exceptions are configured to raise on wait_fors`